spacy/cli/debug_data.py

```diff
@@ -193,6 +194,65 @@ def debug_data(
     else:
         msg.info("No word vectors present in the package")
 
+    if "spancat" in factory_names:
+        model_labels_spancat = _get_labels_from_spancat(nlp)
+        has_low_data_warning = False
+        has_no_neg_warning = False
+
+        msg.divider("Span Categorization")
+        msg.table(model_labels_spancat, header=["Spans Key", "Labels"], divider=True)
+
+        msg.text("Labels in train data: ")
+        for spans_key, data_labels in gold_train_data["spancat"].items():
+            msg.text(
+                f"Key: {spans_key}, {_format_labels(data_labels.items(), counts=True)}"
+            )
+        # Actual data checks
+        for spans_key, data_labels in gold_train_data["spancat"].items():
+            for label, count in data_labels.items():
```

Comment:
Would it make sense to do this just for the spans keys in actual spancat components? I think we risk showing warnings related to irrelevant spans keys in the reference data otherwise, for instance, the "low number of examples" warning would be shown for all spans keys, I think?

Reply:
Gotcha, I filtered the `gold_train_data["spancat"]` with the relevant `spans_key` from `_get_labels_from_spancat` here: https://github.com/explosion/spaCy/pull/10070/commits/21c6351c63702a7a07afc51fb2b6f3e0c70ea412 (stored in the `model_labels_spancat` variable).